Lesson2/Task9.cs

```diff
@@ -0,0 +1,42 @@
+using Microsoft.VisualBasic;
+
+namespace CSharpFundamentals.Lesson2;
+
+class Task9
+{
+    public void CalculateAmountBankTests()
+    {
+        try
+        {
+            Console.WriteLine("Calculate the amount in the bank account with anual interest.");
+
+            Console.Write("Enter amount: ");
+            double amount = Convert.ToDouble(Console.ReadLine());
+
+            Console.Write("Enter the annual percentage rate: ");
+            double percentage = Convert.ToDouble(Console.ReadLine());
+
+            Console.Write("Enter year: ");
+            int year = Convert.ToInt32(Console.ReadLine());
+
+            double result = CalculateAmountBank(amount, percentage, year);
+            Console.WriteLine("Amount in the bank at annual interest: " + result);
+        }
+
+        catch (Exception e)
+        {
+            Console.WriteLine(e);
+        }
+    }
+
+    public double CalculateAmountBank(double amount, double percentage, int year)
+    {
+        if (amount <= 0 || percentage <= 0 || year <= 0)
```

Comment:
Separate conditions by amount, percentage, year and give them specific messages.

Reply:
Fixed